src/launch/github/generate_github_token.py

```diff
@@ -0,0 +1,100 @@
+import sys
+import time
+from pathlib import Path
+
+import boto3
+import requests
+from botocore.exceptions import ClientError
+from jwt import PyJWT
+
+
+def create_jwt(
+    application_id: int, private_key: bytes, expiration_seconds: int = 600
+) -> str:
+    payload = {
+        "iat": int(time.time()),
+        "exp": int(time.time()) + expiration_seconds,
+        "iss": application_id,
+    }
+    jwt_instance = PyJWT()
+    encoded_jwt = jwt_instance.encode(
+        payload=payload, key=private_key, algorithm="RS256"
+    )
+    return encoded_jwt
+
+
+def get_token(
+    application_id_parameter_name: str,
+    installation_id_parameter_name: str,
+    signing_cert_secret_name: str,
+) -> str:
+    application_id = get_ssm_parameter(application_id_parameter_name)
+    installation_id = get_ssm_parameter(installation_id_parameter_name)
+    signing_cert_secret_cm_name = get_ssm_parameter(signing_cert_secret_name)
+
+    signing_jwt = create_jwt(
+        application_id=application_id,
+        private_key=get_secret_value(signing_cert_secret_cm_name),
+    )
+    headers = {"Authorization": f"Bearer {signing_jwt}"}
+    response = requests.post(
+        url=f"https://api.github.com/app/installations/{installation_id}/access_tokens",
+        headers=headers,
+    )
+    return response.json()["token"]
+
+
+def get_ssm_parameter(parameter_name):
```

Comment:
Please ensure this and other functions have annotations on their arguments and an annotation for their return type if any.

```suggestion
def get_ssm_parameter(parameter_name: str) -> str:
```